### DISCUSSION TITLE
What is the unit of measurement for frequency?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#753] [Galaxy Trivia] [answered]
**Bonus:** What is the Earth's resonant frequency?

**Comment by SANKARAMDAS** [comment]
hertz (Hz)
The SI unit for frequency is the hertz (Hz).

Earth's resonant frequency - about 8 hertz
The primary resonance is at a frequency of about 8 hertz—eight trips around the planet per second. The resonance is maintained by the combined effect of all the lightning flashes on Earth—between 40 and 50 per second.

**Comment by peterbaccaro** [comment, marked as answer]
The unit of measurement for frequency is **Hertz (Hz)**, which represents the number of cycles or oscillations per second. It is named after the German physicist Heinrich Hertz, who was the first to demonstrate the existence of electromagnetic waves. The Hertz is commonly used to measure various types of waves, including sound waves, radio waves, and light waves.

The Earth's resonant frequency is approximately **7.83 Hz**, which is also known as the Schumann resonance. This frequency was first predicted by the German physicist W.O. Schumann in 1952, and later measured by him and H.L. König in 1954. The Schumann resonance is created by the electrical discharge activity in the atmosphere between the Earth's surface and the ionosphere, and is influenced by lightning strikes, solar radiation, and other natural factors. It is also believed to have an effect on human brain activity and consciousness.
